load-test-framework/dotnet_src/Google.PubSub.Flic/Program.cs

```diff
@@ -0,0 +1,225 @@
+﻿using CommandLine;
```

Comment:
Please split this up by moving each static class into their own file.  In addition to getting load test results, we'd like to have a relatively short code sample showing users how to configure clients for performance in each language.

Reply:
done